src/BugsnagServiceProvider.php

```diff
@@ -28,27 +29,52 @@ class BugsnagServiceProvider extends ServiceProvider
      */
     public function boot()
     {
+        $this->app->call([$this, 'setupConfig']);
```

Comment:
I'm curious, why are you calling them like this instead of `$this->setupConfig()`?